mlflow/store/file_store.py

```diff
@@ -432,14 +432,22 @@ def log_metric(self, run_uuid, metric):
         make_containing_dirs(metric_path)
         append_to(metric_path, "%s %s\n" % (metric.timestamp, metric.value))
 
+    def _writeable_value(self, tag_value):
+        if tag_value is None:
+            return ""
```

Comment:
Sorry I missed this earlier, but just trying to understand -- why might None get passed here? It's probably ok to write it as "" if users do set_tag(name, None) but maybe we should make that mean to delete the tag or something like that. It's kind of confusing because this is on the server side anyway; does the public API even allow set_tag(name, None)?

Reply:
Yes user can explicitly call `mlflow.set_tag("blah", None)`, to store a single piece of information not key-value. If we dont convert it to "", it will show up as "None" in value.

If we need to support delete, IMHO we should have an explicit `unset_tag`.

Reply:
Ah, that makes sense, I guess we should convert it to "" in that case.

Reply:
Cool! So the way this is written is that `None` will get converted to `""` before getting written to tag file.